README.md

```diff
@@ -1 +1,51 @@
-# registry-zookeeper
\ No newline at end of file
+# registry-zookeeper
```

Comment:
You may also specify this repo is mainly developed by you<Github ID> in the end of this file

Reply:
fine